simpleflow/canvas.py

```diff
@@ -58,7 +58,9 @@ def __init__(self,
 
     def append(self, submittable, *args, **kwargs):
         from simpleflow import Workflow
-        if isinstance(submittable, (Submittable, SubmittableContainer)):
+        if isinstance(submittable, WorkflowTask):
```

Comment:
I'd put this case in the `if isinstance(submittable, (Submittable, SubmittableContainer)):` branch, after the args/kwargs check, for robustness